### PR TITLE
Replace template with actual implementation

### PR DESCRIPTION
**Pull request by Tar-Dingens**

### CODE REVIEW

**Comment by jiri-janousek** [comment]
Hi. Will take a look at it in July 2015. tiliado/nuvolaplayer.project#4

File: metadata.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
     "id": "logitech_media_server",
     "name": "Logitech Media Server",
-    "maintainer_name": "Steffen Coenen?",
-    "maintainer_link": "https://github.com/FIXME",
+    "maintainer_name": "Steffen Coenen",
+    "maintainer_link": "https://github.com/Tar-Dingens",
     "version_major": 2,
-    "version_minor": 0,
+    "version_minor": 1,
```
Comment (jiri-janousek) [inline metadata.json]: Leave version_minor 0 as this will be the first public release.

File: integrate.js [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+	var state = PlaybackState.UNKNOWN
+	try {
+		state = ((document.getElementById("ctrlTogglePlay").firstChild.className.indexOf("btn-play")>-1))? PlaybackState.PAUSED : PlaybackState.PLAYING;
+		}
+	catch (e) {
+		state = PlaybackState.UNKNOWN
+	} 
+
+	//Set state to unknown if there is no available song
+	if(state==!track.title){
```
Comment (jiri-janousek) [inline integrate.js]: You probably wanted `if(!track.title)`.

**Comment by jiri-janousek** [comment]
Hello @Tar-Dingens. Code looks well. I've only found a few minor issues (see inline comments). There are some commit messages in German. Could you use only English next time? Thanks :-)

Could you also update the Makefile to the recent template and add "src/icon.svg"? You can download required files from [the repository](https://github.com/tiliado/nuvolaplayer/tree/master/web_apps/test) or you can find them in Nuvola Player 3 installation:
- /usr/share/nuvolaplayer3/web_apps/test/Makefile
- /usr/share/nuvolaplayer3/web_apps/test/src/icon.svg

If you use the generic icon mentioned above, also add following copyright information to README.md:

```
  File src/icon.svg
  Copyright 2011 Arturo Torres Sánchez <arturo.ilhuitemoc@gmail.com>
  Copyright 2011 Alexander King
  License: CC-BY-3.0
```

**Comment by Tar-Dingens** [comment]
Hello @fenryxo. Everything should be fine now.

**Comment by jiri-janousek** [comment]
Merged. Will create packages within an hour. Welcome in the Nuvola Player team :-)
- Subscribe to the [Nuvola Player Development forum](https://groups.google.com/d/forum/nuvola-player-devel) as I post here announcements related to development.
- Create an account at [Tiliado.eu](https://tiliado.eu) and send me your username. I'll add you to Tiliado developers group. You can then fill in your credentials in Nuvola Player preferences not to be bothered by donate bar.

**Pull request merged**